BudgetSquirrel.Api/ClientApp/src/app/splash-page/splash-page.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="splash">
```

Comment:
Ahhh. well, I think that smaller components are good... if they truely are reusable. The splash page, however, isn't suppose to be re-usable. It's a specific page. It has a very specific purpose that only fulfills the splash page.

Also, I can see how it might make it annoying to flip back and fourth between files, but in my experience, I've never even been bothered by flipping back and fourth cause it's never been that much of an issue. Plus, this gives us a better separation of concerns which, in my opinion, is worth some extra flipping back and fourth.